eogrow/core/pipeline.py

```diff
@@ -171,8 +171,11 @@ def run_execution(
         executor_class: type[EOExecutor]
 
         execution_kind = self._init_processing()
+        extra_kwargs = {}
         if execution_kind is ProcessingType.RAY:
             executor_class = RayExecutor
+            if self.config.worker_type is not None:
+                extra_kwargs = {"ray_remote_kwargs": {"resources": {self.config.worker_type: 0.001}}}
```

Comment:
Do we need to check how this affects scheduling? i.e. when the cluster is empty (head only) ray starts creating new nodes based on the amount of tasks in the queue, and by default this takes into account how many CPUs there are on the machine (or how many were specified to be used by the resources tag). My concern here would be that this means that the scheduler would think "OK, I can only create 1 new node, since that covers the 1000 cases", while in fact this would then represent a bottleneck, because the new node only has e.g. 8 CPUs

Reply:
Every remote worker still requests 1 CPU by default. If you are worried about this we can add this explicitly in eo-learn.

Reply:
So just to make clear, `options` only override the parameters that were given. Since we do not pass new requirements for CPUs they stay at 1.
![image](https://github.com/sentinel-hub/eo-grow/assets/31988337/df7a3b97-39da-490f-a899-ab724bcb4dd0)